ArduCopter/GCS_Mavlink.cpp

```diff
@@ -582,6 +582,9 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_packet(const mavlink_command_i
     switch(packet.command) {
     case MAV_CMD_DO_FOLLOW:
 #if MODE_FOLLOW_ENABLED == ENABLED
+        if (isnan(packet.param1) || isinf(packet.param1)) {
```

Comment:
Why bother checking this? You can't get into the accepted path here as one or both of the comparisons will fail, this just adds a different return value for a rare case, I don't think it's worth the runtime checks and code space.

Reply:
@WickedShell san.
Thanks for your comment. I consider using AP_Mission :: sanity_check_params ().